src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.domain;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> getCarList() {
```

Comment:
변수나 메서드명 뒤에 자료형 `List`이 붙어있네요! 
```suggestion
    public List<Car> getCars() {
```